### PR TITLE
ADCM-2453

### PR DESCRIPTION
**Pull request by kuhella**
fixed duplicates, roles category, naming convention, linked roles base on bundle action to built-in role type roles (Service Administrator etc)

### CODE REVIEW

File: python/cm/bundle.py [hunk 1]
```diff
@@ -299,32 +306,52 @@ def cook_roles(bundle):
                     'prototype__bundle_id': bundle.id,
                 },
             },
+            parametrized_by_type=act.prototype.type,
```
Comment (a-alferov) [inline python/cm/bundle.py:309]: `parametrized_by_type` is a `list`

File: python/cm/bundle.py [hunk 2]
```diff
@@ -265,11 +265,18 @@ def update_obj(dest, source, fields):
         setattr(dest, f, getattr(source, f))
 
 
+def extend_role(name, children):
+    role = Role.objects.get(name=name)
+    for child_role in children:
```
Comment (a-alferov) [inline python/cm/bundle.py:270]: We don't need a loop here, we can add all the children like this `role.child.add(*children)`

**Review by a-alferov — CHANGES_REQUESTED** [review]

File: python/cm/bundle.py (hunk 1 above)
Comment (kuhella) [reply python/cm/bundle.py:308]: fixed

File: python/cm/bundle.py (hunk 2 above)
Comment (kuhella) [reply python/cm/bundle.py]: fixed

File: python/cm/bundle.py (hunk 1 above)
Comment (nagarin) [reply python/cm/bundle.py:308]: and yes, hanging comma is bad replacement for explicit list literal

File: python/cm/bundle.py [hunk 3]
```diff
@@ -299,32 +305,52 @@ def cook_roles(bundle):
...
         role.permissions.add(perm)
-
-    for name, children in parent.items():
-        role = Role(
-            name=f'{name}',
-            display_name=f'{name}',
-            description=f'action(s) {name}',
-            bundle=bundle,
+        if name not in parent:
+            parent[name] = {'parametrized_by_type': act.prototype.type, 'children': []}
```
Comment (nagarin) [inline python/cm/bundle.py:318]: Should'n parametrized_by_type be here a list too?

File: .gitignore [hunk 4]
```diff
@@ -28,3 +28,4 @@
 /tests/fixture.db
 /web/node_modules/
 /wwwroot
+/run_adcm.sh
```
Comment (a-alferov) [inline .gitignore:31]: remove pls

**Review by a-alferov — CHANGES_REQUESTED** [review]

**Review by nagarin — APPROVED** [review]

**Review by a-alferov — APPROVED** [review]

**Pull request merged**